### DISCUSSION TITLE
is adding exceptions to .nsprc manual only?

### DISCUSSION THREAD

**Discussion by thedukegit** [jeemok/better-npm-audit#45] [Q&A] [answered]
Hi There, 

Not sure if this is the right place to ask this, but I was wondering if adding exceptions to the .nsrpc file is a manual proces only, or are there commands to add the exceptions as well?

Thanks!

**Comment by jeemok** [comment, marked as answer]
Hi @thedukegit, I think this is a better place to discuss and answer matters like this than issues :) 

From `v2.0.4` onwards, you can use the `-x` command option, e.g.:

```bash
better-npm-audit audit -x 567,990
# or
better-npm-audit audit -x "567, 990"
```

> Note: Both exceptions passed from the command or `.nsprc` file will be combined and use together.

More command options are described [here](https://github.com/jeemok/better-npm-audit#options)
